### DISCUSSION TITLE
Problem with  Tailwind CLI build --watch option in Ubuntu 22.04.1 LTS

### DISCUSSION THREAD

**Discussion by zangetsuBankai** [tailwindlabs/tailwindcss#10043] [Help] [answered]
Steps to recreate:

1. put this in your index.html
```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Canva Imitation</title>
    <link href="../dist/output.css" rel="stylesheet" />
</head>
<body>
    <div class="grid grid-cols-4 gap-4">
        <div>01</div>
        <div>02</div>
        <div>03</div>
        <div>04</div>
        <div>05</div>
        <div>06</div>
        <div>07</div>
        <div>08</div>
        <div>09</div>
      </div>
</body>
</html>
```

2. run `npx tailwindcss -i src/input.css -o dist/output.css -w`
3. change `<div class="grid grid-cols-4 gap-4"> in the index.html to <div class="grid grid-cols-3 gap-4">`
4. check the ../dist/output.css file, notice that the "grid-cols-4" class is not deleted.

**Comment by crswll** [comment, marked as answer]
That's by design. If would only remove the usused if you restarted the watch. When you build your site and run it without `-w` it will be good.

**Reply by zangetsuBankai** [reply]
Thanks @crswll, so even in windows or mac, this occurs?

**Reply by crswll** [reply]
Yes.
